minard/views.py

```diff
@@ -546,6 +547,27 @@ def l2():
         return redirect(url_for('l2',step=step,height=height,_external=True))
     return render_template('l2.html',step=step,height=height)
 
+@app.route('/burst')
+def burst():
+    return render_template('burst.html',data=load_burst_runs())
+
+def load_burst_runs():
```

Comment:
This should be added to a new python module. Ideally the views.py file doesn't contain logic to deal with databases, etc. So all the couchdb related stuff should be added to that new file which can then be imported here.

Reply:
Would also be nice to add a comment to this function describing what it does. See https://github.com/snoplus/minard/blob/master/minard/detector_state.py#L594 for a good example.